packages/components/src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+export interface SpinnerProps {
```

Comment:
Unless these props are intended to be used outside of this file/module, there's no need to export them. 
```suggestion
interface SpinnerProps {
```

Reply:
Took away export!